hr_timesheet_sheet_attendance/views/hr_timesheet_sheet_view.xml

```diff
@@ -55,8 +55,8 @@
                                 <button name="attendance_action_change" attrs="{'invisible': ['|', ('attendance_state','=',False), ('attendance_state', '=', 'checked_out')]}" type="object" string="Check Out"  style="margin-left: 20px"/>
                             </div>
                         </group>
-                        <field name="attendances_ids" nolabel="1" context="{'default_employee_id': employee_id}" colspan="4">
-                            <tree editable="bottom" create="false">
+                        <field name="attendances_ids" nolabel="1" options="{'no_create': True, 'no_open': True}" context="{'default_employee_id': employee_id}" readonly="1">
```

Comment:
if it is readonly then why do we need 'no_create': True.
and why do we need to make it read-only? from the sheet normal user can adjust it manually.
especially in the case of the daily sheet. and someone forgets yesterday's signout.
correct me if I am wrong or misunderstood.

Reply:
About the readonly, the question is that Odoo doesn't allow to change attendances (not listing them) if your permission is "Manual attendance". This is a leak for allowing it.

Reply:
True Menu access is not there for the Manual attendance group.